v63005/outref/gtm8962.txt

```diff
@@ -0,0 +1,16 @@
+# Confirming $ZPIN=$PRINCIPAL, searching for $ZPIN in the output of ZSHOW "I"
+YDB>write:($ZPIN=$PRINCIPAL) "# $ZPIN = $PRINCIPAL"
+# $ZPIN = $PRINCIPAL
+$ZPIN="/dev/pts/3"
```

Comment:
I want $PRINCIPAL grepped too. That is the only way we can confirm $ZPIN matches $PRINCIPAL. Same with $ZPOUT.